### ISSUE TITLE
Add Error Log to RobotDiagnostics

### ISSUE THREAD

**Issue by jonathanlew** [UBC-Thunderbots/Software#1287]
### Description of the task
Add Error Log and to RobotDiagnostics, since theses are important debug features

### Acceptance criteria

- [ ] Add Error Log and GUI to RobotDiagnostics
- [ ] Connect it to the rest of the system
- [ ] Manually Test it
### Blocked By
#1228

**Comment by MathewMacDougall** [comment]
Error log is technically already done as it's part of the Visualizer. E-stop indicator should be made generic if possible so it can be shared.

**Comment by jonathanlew** [comment]
Resolved by #1383
